contracts/deploy/Create.sol

```diff
@@ -7,12 +7,12 @@ pragma solidity ^0.8.0;
  * @notice This contract deploys new contracts using the `CREATE` opcode and is used as part of
  * the `Create3` deployment method.
  */
-contract CreateDeployer {
+contract Create {
```

Comment:
I guess CreateDeployer gives the false impression that it supports the same Deployer interface as the others?

Reply:
Exactly